MediaBrowser.Providers/Tmdb/TV/TmdbSeriesProvider.cs

```diff
@@ -39,6 +34,11 @@ public class TmdbSeriesProvider : IRemoteMetadataProvider<Series, SeriesInfo>, I
         private readonly IHttpClient _httpClient;
         private readonly ILibraryManager _libraryManager;
 
+        private readonly CultureInfo _usCulture = new CultureInfo("en-US");
+        private const string GetTvInfo3 = TmdbUtils.BaseTmdbApiUrl + @"3/tv/{0}?api_key={1}&append_to_response=credits,images,keywords,external_ids,videos,content_ratings";
+
+        internal static TmdbSeriesProvider Current { get; private set; }
+
```

Comment:
Why move these, consts should be on top, than readonly fields and properties should come after the constructor